.github/scripts/build_libpng.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+set -e
```

Comment:
Not `-euo pipefail`?

Reply:
`set -u` we can, but `set -o pipefail` is not provided by `sh`. Considering that this is a CI-only script, I guess I'll switch the shebangs to `bash`.